venues/ACM.org/SIGIR/Badging/python/acm19.py

```diff
@@ -156,12 +156,18 @@
                 ],
                 'required': True
             },
+            'html': {
+                'fieldDisplayLabel': 'Html',
+                'description': 'Either provide a direct url to your artifact (link must begin with http(s)) or upload a PDF file',
+                'order': 8,
```

Comment:
Could you put another order number? this is the order how the fields are rendered in the form.

Reply:
Thanks for noticing this. It's fixed.